README.md

```diff
@@ -1 +1,63 @@
-# math-magicians
\ No newline at end of file
+# math magicians
+
+Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations, read a random math-related quote.
+This application is build using React.
+
+
+If you like what you see and have a project you need coded, don't hesitate to contact me.
+
+
+
+# Built With
+
+
+- Major languages: ReactJS,
+- Technologies used: Web services, babelrc
+
+
+# Live Demo
+
+[Live Demo Link]().
+
+
```

Comment:
- [ ] You want to consider removing broken links in your README if you don't have a link for it yet. ♻️